components/webext-storage/src/migration.rs

```diff
@@ -107,23 +108,26 @@ struct Parsed<'a> {
     data: serde_json::Value,
 }
 
-pub fn migrate(tx: &Transaction<'_>, filename: &Path) -> Result<usize> {
+pub fn migrate(tx: &Transaction<'_>, filename: &Path) -> Result<MigrationInfo> {
     // We do the grouping manually, collecting string values as we go.
     let mut last_ext_id = "".to_string();
     let mut curr_values: Vec<(String, serde_json::Value)> = Vec::new();
-    let mut num_extensions = 0;
-    for row in read_rows(filename) {
+    let (rows, mut mi) = read_rows(filename);
+    for row in rows {
         log::trace!("processing '{}' - '{}'", row.col_name, row.record);
         let parsed = match row.parse() {
             Some(p) => p,
-            None => continue,
+            None => {
```

Comment:
how come this change? Did a rustfmt update cause it? It seems a step backwards - I don't want to start hating rustfmt as much as I hate eslint ;)

Reply:
Weird, not 100% sure. Would have expected cargo fmt to normalize that but it doesn't (an earlier verison of the patch did more here)